lib-dmtf/model/Chassis.go

```diff
@@ -86,7 +86,7 @@ type Entries struct {
 
 // Assembly redfish structure
 type Assembly struct {
-	Oid string `json:"@odata.id"`
+	Oid string `json:"@odata.id,omitempty"`
```

Comment:
odata.id is mandatory parameter in Assembly, we shouldn't expect it to be empty.